src/index.ts

```diff
@@ -1,18 +1,31 @@
-/*
- *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *  SPDX-License-Identifier: Apache-2.0
- */
-
-import serverless from 'serverless-http';
-import { generateServerlessRouter } from 'fhir-works-on-aws-routing';
-import { fhirConfig, genericResources } from './config';
-
-const serverlessHandler = serverless(generateServerlessRouter(fhirConfig, genericResources), {
-    request(request: any, event: any) {
-        request.user = event.user;
-    },
-});
-
-export default async (event: any = {}, context: any = {}): Promise<any> => {
-    return serverlessHandler(event, context);
-};
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import serverless from 'serverless-http';
+import { generateServerlessRouter } from 'fhir-works-on-aws-routing';
+import { CorsOptions } from 'cors';
+import { fhirConfig, genericResources } from './config';
+
+const corsOptions: CorsOptions = {
+    origin: [
+        'http://localhost',
+        'https://cdn-fhir-forms-test.optimalcare.com/dwg-form',
+        'https://cdn-fhir-forms.optimalcare.com/dwg-form',
+    ],
```

Comment:
Just making sure: upstream will not accept this. The allowed origins should be configurable from outside.